File: scripts/testes-outras-linguagens/A/CompilationError.go
package main

import (
"fmt"
)

func main() {
var p, ap, tb float64
    
fmt.Scan(&p, &ap, &tb)
    
    // --- ERRO DE COMPILAÇÃO INTENCIONAL ---
    // Tentativa de chamar uma função inexistente no pacote "fmt".
    // Isso deve resultar em um erro do tipo "fmt.FuncaoInexistente undefined".
    fmt.FuncaoInexistente("Isso não deve compilar") 

    // O código de cálculo correto está abaixo, mas não será compilado.
    a1 := (p*7.0 + ap*2.0 + tb*1.0) / 10.0

fmt.Printf("%.3f", a1)
}

File: scripts/testes-outras-linguagens/A/CompilationError.php
<?php

// --- ERRO DE COMPILAÇÃO INTENCIONAL (SINTAXE INVÁLIDA) ---
// O caractere '@' solto fora de uma string ou função não é permitido em PHP.
@

// Erro de sintaxe na palavra-chave (o correto é 'function')
functio main_logic() {
    // Código correto, mas nunca será interpretado
    $input_line = trim(fgets(STDIN));
    $notes = explode(' ', $input_line);
    $P = (float)$notes[0];
    $AP = (float)$notes[1];
    $TB = (float)$notes[2];
    
    $A1 = ($P * 7.0 + $AP * 2.0 + $TB * 1.0) / 10.0;
    
    echo number_format($A1, 3, '.', '') . "\n";
}

main_logic();
?>